app/src/main/java/io/left/ripple/MainActivity.java

```diff
@@ -36,6 +36,7 @@
     // Interface object for the RightMesh library.
     AndroidMeshManager androidMeshManager;
 
+    //update this to your  assigned mesh port
```

Comment:
Minor: too many spaces

Reply:
I wonder if you can suggest on a suggestion. I guess you'd just make a different suggestion in the same thread.
```suggestion
    // Ah, yeah, just puts you into the same line.
```

Reply:
😮